packages/react/src/primitives/Field/Field.tsx

```diff
@@ -50,11 +50,12 @@ const FieldPrimitive: Primitive<FieldPrimitiveProps, typeof Flex> = (
       {...rest}
     >
       {label ? <Label visuallyHidden={labelHidden}>{label}</Label> : null}
-      {descriptiveText ? (
```

Comment:
Wait, why remove this logic? Shouldn't descriptiveText be provided to show the descriptive text?

Reply:
+1

Reply:
That logic is IN `<FieldDescription>`: https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/src/primitives/Field/FieldDescription.tsx#L15

Reply:
but why did that cause a bug?

Reply:
This didn't cause the bug, the bug was I was passing it as a child instead of prop